client/components/App/App.scss

```diff
@@ -8,14 +8,37 @@
 
   position:relative;
 
+  /* monitors and laptops */
+  @media (min-width: 1240px),
+  (min-width: 768px) and (max-width: 1024px) and (orientation: landscape),
+  (max-width: 768px) and (orientation: landscape) {
+    .side-nav {
+      width: calc( 20% + 2em);
+    }
+    .main{
```

Comment:
@IgorChaikin Could you please doublecheck your `prettier` configuration, it should format on save in your ide and change formatting to something like:

```scss
.app {
  width: 100vw;
  height: 100vh;
  display: flex;
  flex-direction: row;

  position: relative;

  /* monitors and laptops */
  @media (min-width: 1240px),
    (min-width: 768px) and (max-width: 1024px) and (orientation: landscape),
    (max-width: 768px) and (orientation: landscape) {
    .side-nav {
      width: calc(20% + 2em);
    }
    .main {
      width: 80%;
    }
  }
  ...
```